src/webui/components/AutoComplete/index.js

```diff
@@ -57,6 +57,27 @@ const renderSuggestion = (suggestion, { query, isHighlighted }): Node => {
   );
 };
 
+const renderMessage = (message): Node => {
+  return (
+    <MenuItem selected={false} component="div">
+      <div>{message}</div>
+    </MenuItem>
+  );
+};
+
+const SUGGESTIONS_RESPONSE = {
```

Comment:
It seems to be useful for other components, perhaps export or move it somewhere else ?

Reply:
I agree with @priscilawebdev 💯